### DISCUSSION TITLE
Sign Up for GitHub Education

### DISCUSSION THREAD

**Discussion by cml1010101** [NorthernForce/Reefscape#3] [Polls]
As students, you all are privy to GitHub Education (essentially GitHub Pro) for free. You additionally keep it through college. Please follow [this link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) to sign up for GitHub education.

**Comment by Daboss-1** [comment]
What school? Falmouth schools? GHS doesn't show up...

**Reply by cml1010101** [reply]
You should be able to write in Gorham High School if Gorham High School is not showing up in a drop-down.

**Reply by Daboss-1** [reply]
It does not pop up. Only GHMS pops up (Gorham High and Middle School), but that isn't us.

**Reply by cml1010101** [reply]
Did you try just typing in Gorham High School? Sometimes you have to make your own entry

**Reply by Daboss-1** [reply]
Oh, yep. Didn't know I could do that. Thank you Connor!

**Comment by Daboss-1** [comment]
I also don't have a 'falmouthschools.org' email account...

**Reply by cml1010101** [reply]
You don't need a falmouthschools.org email. Although this is only for students, the school is not responsible for anything.
